cmd/filegen/main.go

```diff
@@ -477,17 +413,7 @@ func generateFiles(ctx *cli.Context) error {
 			Balance: initialMint,
 		}
 
-		err = core.SaveSkToPemFile(initialBalancesSkFile, pkHex, skHex)
-		if err != nil {
-			return err
-		}
-
-		_, err = initialBalancesSkPlainFile.Write(append(skHex, '\n'))
-		if err != nil {
-			return err
-		}
-
-		_, err = initialBalancesPkPlainFile.Write(append([]byte(pkHex), '\n'))
+		err = core.SaveSkToPemFile(walletKeyFile, pkHex, skHex)
```

Comment:
it is not actually pkHex here, its the bech32. Does whoever reads the walletKeyFile know that there are 2 different formats?

Reply:
the walletApp does not take into consideration the PK. It autogenerates it.